vendor/bat-native-ads/src/bat/ads/internal/base/strings/string_conversions_util.cc

```diff
@@ -18,4 +22,27 @@ std::string BoolToString(const bool value) {
   return value ? kTrue : kFalse;
 }
 
+std::vector<float> ConvertStringToVector(std::string string) {
```

Comment:
So this is reusable. Can we also pass the delimiter, i.e. `" "` and rename the function to ConvertDelimitedStringToVector